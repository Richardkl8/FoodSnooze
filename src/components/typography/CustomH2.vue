<template>
    <h2>
        <slot></slot>
    </h2>
</template>

<script>
    export default {
        name: 'CustomH2',
    };
</script>

<style scoped>
    h2 {
        font-size: 2.5rem;
        color: #071E22;
    }
</style>
