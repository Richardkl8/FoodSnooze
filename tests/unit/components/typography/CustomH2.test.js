import { shallowMount } from '@vue/test-utils';
import CustomH2 from '../../../../src/components/typography/CustomH2.vue';

describe('CustomH2', () => {
    let wrapper;
    beforeEach(() => {
        wrapper = shallowMount(CustomH2, {
            slots: {
                default: 'Test text',
            },
        });
    });

    test('Should match snapshot', () => {
        expect(wrapper).toMatchSnapshot();
    });

    test('Should show text when text is injected in slot', () => {
        expect(wrapper.text()).toMatchSnapshot('Test text');
    });
});
