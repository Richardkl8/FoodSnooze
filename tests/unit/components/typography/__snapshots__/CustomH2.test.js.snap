// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomH2 Should match snapshot 1`] = `<h2>Test text</h2>`;

exports[`CustomH2 Should show text when text is injected in slot: Test text 1`] = `"Test text"`;
